staging/src/k8s.io/client-go/tools/cache/reflector_test.go

```diff
@@ -572,27 +572,24 @@ func TestReflectorFullListIfExpired(t *testing.T) {
 			for i := 0; i < 8; i++ {
 				pods[i] = v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: fmt.Sprintf("pod-%d", i), ResourceVersion: fmt.Sprintf("%d", i)}}
 			}
-			switch options.ResourceVersion {
-			case "0":
+			rv_continue_limit := func(rv, c string, l int64) metav1.ListOptions {
```

Comment:
lint was complaining about the underscores in this name, sorry

Reply:
Done.